recipes/enchant/all/patches/0002-remove-strchrnul-usage.patch

```diff
@@ -0,0 +1,33 @@
+--- src/lib.c
++++ src/lib.c
+@@ -177,12 +177,17 @@
+ enchant_normalize_dictionary_tag (const char * const dict_tag)
+ {
+ 	char * new_tag = g_strstrip (strdup (dict_tag));
++	char * tmp;
+ 
+ 	/* strip off en_GB@euro */
+-	*strchrnul (new_tag, '@') = '\0';
```

Comment:
```
The strchrnul() function is like strchr() except that if c is not found in s, then it returns a pointer to the null byte at the end of s, rather than NULL.
```
patch doing an equivalent behavior